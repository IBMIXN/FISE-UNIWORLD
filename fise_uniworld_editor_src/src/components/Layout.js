import PropTypes from 'prop-types';
import React from 'react';
import { CssBaseline } from '@material-ui/core';
import Navbar from './Navbar';

const Layout = ({ children }) => (
  <React.Fragment>
    <CssBaseline />
    <Navbar />
    {children}
    <div style={{ marginBottom: '50px' }} />
  </React.Fragment>
);

Layout.propTypes = {
  children: PropTypes.node,
};

export default Layout;
