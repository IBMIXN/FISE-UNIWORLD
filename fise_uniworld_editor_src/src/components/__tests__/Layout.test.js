import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../Layout';

describe('Layout test', () => {
  it('Layout should match snapshot', () => {
    const component = shallow(<Layout />);
    expect(component).toMatchSnapshot();
  });
});
