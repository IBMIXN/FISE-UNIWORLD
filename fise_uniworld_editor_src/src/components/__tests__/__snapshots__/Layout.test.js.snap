// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout test Layout should match snapshot 1`] = `
<Fragment>
  <WithStyles(CssBaseline) />
  <Navbar />
  <div
    style={
      Object {
        "marginBottom": "50px",
      }
    }
  />
</Fragment>
`;
